<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task</class>
 <widget class="QWidget" name="Task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collection Tasks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/img/tasks.svg</normaloff>:/img/img/tasks.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="wrapper">
     <property name="styleSheet">
      <string notr="true">QFrame#wrapper {
	background-color: rgb(243, 246, 253);
	border-radius: 30px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins Black</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/img/img/close.svg</normaloff>:/img/img/close.svg</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>821</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#title{
	color: rgb(60, 61, 81);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="task1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(238, 240, 253);
border-radius: 20px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>9</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/remove.svg</normaloff>:/img/img/remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="tasks">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dosis ExtraBold</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(60, 61, 81);</string>
      </property>
      <property name="text">
       <string>Tasks</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="task2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>261</y>
        <width>461</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(238, 240, 253);
border-radius: 20px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/remove.svg</normaloff>:/img/img/remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="task3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>321</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(238, 240, 253);
border-radius: 20px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/remove.svg</normaloff>:/img/img/remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="task4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>382</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(238, 240, 253);
border-radius: 20px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/remove.svg</normaloff>:/img/img/remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="task5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>444</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(238, 240, 253);
border-radius: 20px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/remove.svg</normaloff>:/img/img/remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="task6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>507</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(238, 240, 253);
border-radius: 20px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>421</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/remove.svg</normaloff>:/img/img/remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>31</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="addframe">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#addframe{background: rgb(243, 246, 253);
border-radius: 20px;
border: 1px solid rgb(85, 170, 255);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="img.qrc">:/img/img/add.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="addline">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>11</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
background: rgb(243, 246, 253);</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Add more task...</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>688</x>
        <y>95</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    border: 1px solid rgb(85, 170, 255);
	background: rgb(243, 246, 253);
}</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
